tests/unit/pandas_utils/test_lusid_pandas.py

```diff
@@ -157,3 +157,68 @@ def test_response_to_df_format(self):
         self.assertIn("PortionClass(MultiAssetScope-SubHoldingKeys)", df_columns)
         self.assertIn("shares_out(MultiAssetScope-Properties)", df_columns)
         self.assertNotIn("JunkBadColumn(SubHoldingKey)", df_columns)
+
+    def test_response_to_df_rename_mapping_format(self):
+
+        rename_mapping = {
+            "instrument_uid": "Instrument ID",
+            "holding_type": "Holding Type",
+            "units": "Units",
+            "settled_units": "Settled Units",
+            "cost.amount": "Quantity",
+            "cost.currency": "Cost Currency",
+        }
+
+        test_holdings_response = lusid.models.versioned_resource_list_of_portfolio_holding.VersionedResourceListOfPortfolioHolding(
+            version=1, values=[self.holding_1]
+        )
+
+        holdings_df = lusid_response_to_data_frame(
+            test_holdings_response,
+            rename_properties=True,
+            rename_mapping=rename_mapping,
+        )
+
+        df_columns = holdings_df.columns
+
+        self.assertIsInstance(holdings_df, pd.DataFrame)
+        self.assertEqual(holdings_df.loc[0]["Instrument ID"], "LUID_123")
+        self.assertIn("PortionClass(MultiAssetScope-SubHoldingKeys)", df_columns)
+        self.assertTrue(
+            set(
+                ["Holding Type", "Units", "Settled Units", "Quantity", "Cost Currency"]
+            ).issubset(set(df_columns))
+        )
+
+    def test_response_to_df_rename_mapping_no_format(self):
+
+        rename_mapping = {
+            "instrument_uid": "Instrument ID",
+            "holding_type": "Holding Type",
+            "units": "Units",
+            "settled_units": "Settled Units",
+            "cost.amount": "Quantity",
+            "cost.currency": "Cost Currency",
+        }
+
+        test_holdings_response = lusid.models.versioned_resource_list_of_portfolio_holding.VersionedResourceListOfPortfolioHolding(
+            version=1, values=[self.holding_1]
+        )
+
+        holdings_df = lusid_response_to_data_frame(
+            test_holdings_response,
+            rename_properties=False,
+            rename_mapping=rename_mapping,
+        )
+
+        df_columns = holdings_df.columns
+
+        print(df_columns)
+
+        self.assertIsInstance(holdings_df, pd.DataFrame)
```

Comment:
as above

Reply:
Thats done.
Updated to `self.assertEquals`